### DISCUSSION TITLE
feature request: `.prettierignore` for subfolders, like `.gitignore`

### DISCUSSION THREAD

**Discussion by trusktr** [prettier/prettier#15143] [Ideas]
- Right now, it is not possible to place a `.prettierignore` file in a subfolder and have it apply to files in that subfolder, akin to how `.gitignore` works.
- Using `--ignore-path` is cumbersome to maintain, and personally I couldn't get it to work.
- Prettier config files do not have an `ignorePath` option to configure this there instead of in the CLI command.

It would be _great_ if we could simply place `.prettierignore` files at any level deep, and for it to just work just like `.gitignore` does (for a given file, see get ignore files up the tree, and see if it is ignored by any of them).

Currently Prettier only uses `./.prettierignore` in the working directly.

**Comment by ELowry** [comment]
This would be a massive benefit for larger projects with multiple independent parts that may be maintained by different people, or in different environments.
